Source/WTF/wtf/OptionSet.h

```diff
@@ -149,11 +149,23 @@ template<typename E> class OptionSet {
         }
     }
 
+    constexpr OptionSet(std::initializer_list<OptionSet> initializerList)
+    {
+        for (auto& optionSet : initializerList)
+            add(optionSet);
+    }
+
     constexpr OptionSet(std::optional<E> optional)
         : m_storage(optional ? static_cast<StorageType>(*optional) : 0)
     {
     }
 
+    constexpr OptionSet& operator=(std::nullptr_t)
```

Comment:
Why is this overload required? Are `std::initializer_list` considered as pointer types?